src/sentry/snuba/metrics/query_builder.py

```diff
@@ -426,6 +426,29 @@ def parse_query(query_string: str, projects: Sequence[Project]) -> Sequence[Cond
     return where
 
 
+def parse_query_params(query_params, projects: Sequence[Project]) -> Sequence[Condition]:
+    try:
+        query_builder = ReleaseHealthQueryBuilder(
+            Dataset.Sessions,
+            params={
+                **query_params,
+                "organization_id": org_id_from_projects(projects) if projects else None,
+            },
+            config=QueryBuilderConfig(use_aggregate_conditions=True),
+        )
+
+        param_conditions = list(
```

Comment:
What tests were failing without this?

Reply:
There are assertion errors in the following tests without this filter:
* `test_build_snuba_query_mri`
* `test_build_snuba_query_derived_metrics`
* `test_build_snuba_query_orderby`
* `test_build_snuba_query_with_derived_alias`

And the failure message shows there's an extra condition for `project_id` being added when one was previously added already:
```
...
E     Matching attributes:
E     ['match',
E      'select',
E      'groupby',
E      'array_join',
E      'having',
E      'orderby',
E      'limitby',
E      'limit',
E      'offset',
E      'granularity',
E      'totals']
E     Differing attributes:
E     ['where']
E     
E     Drill down into differing attribute where:
E       where: [Condition(lhs=Column(name='org_id', entity=None, subscriptable=None, key=None), op=<Op.EQ: '='>, rhs=1), Condition(lhs=Column(name='project_id', entity=None, subscriptable=None, key=None), op=<Op.IN: 'IN'>, rhs=[1]), Condition(lhs=Column(name='timestamp', entity=None, subscriptable=None, key=None), op=<Op.GTE: '>='>, rhs=datetime.datetime(2021, 8, 24, 23, 0, tzinfo=datetime.timezone.utc)), Condition(lhs=Column(name='timestamp', entity=None, subscriptable=None, key=None), op=<Op.LT: '<'>, rhs=datetime.datetime(2021, 8, 26, 0, 0, tzinfo=datetime.timezone.utc)), Condition(lhs=Column(name='tags[9223372036854776016]', entity=None, subscriptable='tags', key='9223372036854776016'), op=<Op.IN: 'IN'>, rhs=[9223372036854776018]), Condition(lhs=Column(name='project_id', entity=None, subscriptable=None, key=None), op=<Op.IN: 'IN'>, rhs=[]), Condition(lhs=Column(name='metric_id', entity=None, subscriptable=None, key=None), op=<Op.IN: 'IN'>, rhs=[9223372036854775812])] != [Condition(lhs=Column(name='org_id', entity=None, subscriptable=None, key=None), op=<Op.EQ: '='>, rhs=1), Condition(lhs=Column(name='project_id', entity=None, subscriptable=None, key=None), op=<Op.IN: 'IN'>, rhs=[1]), Condition(lhs=Column(name='timestamp', entity=None, subscriptable=None, key=None), op=<Op.GTE: '>='>, rhs=datetime.datetime(2021, 8, 24, 23, 0, tzinfo=datetime.timezone.utc)), Condition(lhs=Column(name='timestamp', entity=None, subscriptable=None, key=None), op=<Op.LT: '<'>, rhs=datetime.datetime(2021, 8, 26, 0, 0, tzinfo=datetime.timezone.utc)), Condition(lhs=Column(name='tags[9223372036854776016]', entity=None, subscriptable='tags', key='9223372036854776016'), op=<Op.IN: 'IN'>, rhs=[9223372036854776018]), Condition(lhs=Column(name='metric_id', entity=None, subscriptable=None, key=None), op=<Op.IN: 'IN'>, rhs=[9223372036854775812])]
E       At index 5 diff: Condition(lhs=Column(name='project_id', entity=None, subscriptable=None, key=None), op=<Op.IN: 'IN'>, rhs=[]) != Condition(lhs=Column(name='metric_id', entity=None, subscriptable=None, key=None), op=<Op.IN: 'IN'>, rhs=[9223372036854775812])
E       Left contains one more item: Condition(lhs=Column(name='metric_id', entity=None, subscriptable=None, key=None), op=<Op.IN: 'IN'>, rhs=[9223372036854775812])
E       Full diff:
E         [
E          Condition(lhs=Column(name='org_id', entity=None, subscriptable=None, key=None), op=<Op.EQ: '='>, rhs=1),
E          Condition(lhs=Column(name='project_id', entity=None, subscriptable=None, key=None), op=<Op.IN: 'IN'>, rhs=[1]),
E          Condition(lhs=Column(name='timestamp', entity=None, subscriptable=None, key=None), op=<Op.GTE: '>='>, rhs=datetime.datetime(2021, 8, 24, 23, 0, tzinfo=datetime.timezone.utc)),
E          Condition(lhs=Column(name='timestamp', entity=None, subscriptable=None, key=None), op=<Op.LT: '<'>, rhs=datetime.datetime(2021, 8, 26, 0, 0, tzinfo=datetime.timezone.utc)),
E          Condition(lhs=Column(name='tags[9223372036854776016]', entity=None, subscriptable='tags', key='9223372036854776016'), op=<Op.IN: 'IN'>, rhs=[9223372036854776018]),
E       +  Condition(lhs=Column(name='project_id', entity=None, subscriptable=None, key=None), op=<Op.IN: 'IN'>, rhs=[]),
E          Condition(lhs=Column(name='metric_id', entity=None, subscriptable=None, key=None), op=<Op.IN: 'IN'>, rhs=[9223372036854775812]),
E         ]
```